src/Feature/Search/Feature_Search.re

```diff
@@ -31,8 +31,8 @@ let initial = {
 
 module Configuration = {
   open Config.Schema;
-
   let searchExclude = setting("search.exclude", list(string), ~default=[]);
+  let filesExclude = setting("files.exclude", list(string), ~default=["_esy",".git","node_modules"]);
```

Comment:
I dont know if this the right thing to do, but its working

Reply:
Great idea! Makes sense to define both of these settings here.